exporters/ostream/include/opentelemetry/exporters/ostream/span_exporter.h

```diff
@@ -110,11 +110,15 @@ class OStreamSpanExporter final : public sdktrace::SpanExporter
     size_t i    = 1;
     for (auto kv : map)
     {
-      sout_ << kv.first << ": ";
+      if (i == 1)
+      {
+        sout_ << std::endl;
+      }
```

Comment:
I think we could move this out of the loop and get rid of `i` variable completely with small change in `Export` function.

Reply:
Good point. Get rid of this variable now.